doc/site/core/sequence.markdown

```diff
@@ -18,6 +18,18 @@ If it returns `false`, stops iterating and returns `false`. Otherwise, returns
     [1, 2, 3].all {|n| n > 2} // False.
     [1, 2, 3].all {|n| n < 4} // True.
 
+### **any**(predicate)
+
+Tests whether any element in the sequence passes the `predicate`.
+
+Iterates over the sequence, passing each element to the function `predicate`.
+If it returns `true`, stops iterating and returns `true`. Otherwise, returns
```

Comment:
Hmm, it does, but only because `1` is considered trueish. You'd have to write `[1, 2, 3].any { |n| n == 1 }` or `[1, 2, 3].contains(1)` to get what you want. This does not represent a problem with this patch, right?